### PR TITLE
[Doc] Update golobal dict

### PR DESCRIPTION
**Pull request by hellolilyliuyi**
## What I'm doing:

Fixes #issue

## What type of PR is this:

- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [ ] Yes, this PR will result in a change in behavior.
- [x] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function
- [ ] This is a backport pr

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [x] 3.3
  - [x] 3.2
  - [ ] 3.1
  - [ ] 3.0
  - [ ] 2.5

### CODE REVIEW

File: docs/zh/using_starrocks/query_acceleration_with_auto_increment.md [hunk 1]
```diff
@@ -367,26 +160,15 @@ SELECT id, COUNT(DISTINCT order_uuid) FROM dest_table_pk GROUP BY id ORDER BY id
...
-        ```
+2. 向聚合表 `dest_table_bitmap` 的 `id` 列插入表 `dest_table` 的列 `id` 的数据；向聚合表的 `order_id_bitmap` 列插入字典表 `dict` INTEGER 列 `order_id_int` 列的数据（经过函数 `to_bitmap` 处理后的值）。
+
+      ```SQL
+      INSERT INTO dest_table_bitmap (id, order_id_bitmap)
+      SELECT dest_table.id, to_bitmap(dict.order_id_int)
+      FROM dest_table LEFT JOIN dict
+          ON dest_table.order_uuid = dest_table.order_uuid
+      WHERE dest_table.batch = 1; -- 用来模拟导入的一批批数据，导入第二批数据时指定为 2
+      ```
```
Comment (hellolilyliuyi) [inline docs/zh/using_starrocks/query_acceleration_with_auto_increment.md:171]: 这里可以使用dict_mapping吗

File: docs/zh/using_starrocks/query_acceleration_with_auto_increment.md [hunk 2]
```diff
@@ -266,88 +104,43 @@ SELECT id, COUNT(DISTINCT order_uuid) FROM dest_table GROUP BY id ORDER BY id;
...
 
-**阶段二：创建目标表并导入 CSV 文件的订单数据。**
-
-需要注意在本解决方案中，目标表必须为主键表。因为只有主键表支持带有 Join 的 UPDATE 命令。
+**阶段二**：创建目标表，并且包含具有 `dict_mapping` 属性的字典 ID 列，后续导入订单数据至目标表时，系统将自动关联字典表并插入对应的字典 ID。
 
-1. 创建一个**主键表** `dest_table_pk`，包含 CSV 文件的所有列。 并且您还需要定义一个 INTEGER 类型的 `order_id_int` 列，与 STRING 类型的 `order_id_int` 列进行映射。后续会基于 `order_id_int` 列进行查询分析。
-
-   并且值得注意的是，为了将所有导入的数据到存在主键表中，您还需要定义一个自增列 `implicit_auto_id` 为主键，作为每一条数据的唯一标识，以确保保留所有数据。（如果实际订单业务表中已经存在其他主键，则直接使用即可，不再需要创建 `implicit_auto_id`）。
+1. 创建一张表 `dest_table`，包含 CSV 文件的所有列。 并且您还需要定义一个 INTEGER 类型的 `order_id_int` 列，与 STRING 类型的 `order_id_int` 列进行映射，并且具有 dict_mapping 列属性。后续会基于 `order_id_int` 列进行查询分析。
```
Comment (jaogoy) [inline docs/zh/using_starrocks/query_acceleration_with_auto_increment.md:109]: INTEGER 类型的 `order_id_int` 列。
这里的INTEGER 是个通用的意思，其实精确的是 BIGINT 类型，不知道有没有影响。

File: docs/zh/using_starrocks/query_acceleration_with_auto_increment.md [hunk 3]
```diff
@@ -266,88 +104,43 @@ SELECT id, COUNT(DISTINCT order_uuid) FROM dest_table GROUP BY id ORDER BY id;
           -XPUT http://<fe_host>:<fe_http_port>/api/example_db/dict/_stream_load
       ```
 
-**阶段二：创建目标表并导入 CSV 文件的订单数据。**
+**阶段二**：创建目标表，并且包含具有 `dict_mapping` 属性的字典 ID 列，后续导入订单数据至目标表时，系统将自动关联字典表并插入对应的字典 ID。
```
Comment (srlch) [inline docs/zh/using_starrocks/query_acceleration_with_auto_increment.md:107]: 阶段1，2之间是否需要说明，如果有增量数据进来，需要在使用dict_mapping之前先往字典表导入一次保证映射一定存在？

File: docs/zh/using_starrocks/query_acceleration_with_auto_increment.md [hunk 4]
```diff
@@ -160,16 +164,58 @@ SELECT id, COUNT(DISTINCT order_uuid) FROM dest_table GROUP BY id ORDER BY id;
...
       ```
 
-2. 向聚合表 `dest_table_bitmap` 的 `id` 列插入表 `dest_table` 的列 `id` 的数据；向聚合表的 `order_id_bitmap` 列插入字典表 `dict` INTEGER 列 `order_id_int` 列的数据（经过函数 `to_bitmap` 处理后的值）。
+2. 向聚合表 `dest_table_bitmap` 中插入数据。 `id` 列插入表 `dest_table` 的列 `id` 的数据；`order_id_bitmap` 列插入字典表 `dict` INTEGER 列 `order_id_int` 的数据（经过函数 `to_bitmap` 处理后的值）。
+
+    ```SQL
+    INSERT INTO dest_table_bitmap (id, order_id_bitmap)
+    SELECT id,  to_bitmap(dict_mapping('dict', order_uuid))
+    FROM dest_table
+    WHERE dest_table.batch = 1;
```
Comment (jaogoy) [inline docs/zh/using_starrocks/query_acceleration_with_auto_increment.md:173]: 这里的batch也加个注释，说是用来模拟不同批次的处理

File: docs/zh/using_starrocks/query_acceleration_with_auto_increment.md [hunk 5]
```diff
@@ -160,16 +164,58 @@ SELECT id, COUNT(DISTINCT order_uuid) FROM dest_table GROUP BY id ORDER BY id;
       DISTRIBUTED BY HASH(id) BUCKETS 2;
```
Comment (jaogoy) [inline docs/zh/using_starrocks/query_acceleration_with_auto_increment.md:164]: 这个bucket 2改为 `BUCKETS 6` 吧。
（因为这个是非分区表，自动设置可能有问题）

Comment (jaogoy) [reply docs/zh/using_starrocks/query_acceleration_with_auto_increment.md:164]: 下同

File: docs/zh/using_starrocks/query_acceleration_with_auto_increment.md [hunk 6]
```diff
@@ -160,16 +164,58 @@ SELECT id, COUNT(DISTINCT order_uuid) FROM dest_table GROUP BY id ORDER BY id;
...
       ```
 
+2. 向聚合表中插入数据。`id` 列直接插入 CSV 文件中的`id` 列的数据；`order_id_bitmap` 列插入字典表 `dict` INTEGER 列 `order_id_int` 列的数据（经过函数 `to_bitmap` 处理后的值）。
+
+     ```bash
+     curl --location-trusted -u root: \
+         -H "format: CSV" -H "column_separator:," \
+         -H "columns: id, order_uuid,  order_id_bitmap=to_bitmap(dict_mapping('dict', order_uuid))" \
+         -T batch1.csv \
+         -XPUT http:///<fe_host>:<fe_http_port>/api/example_db/dest_table_bitmap/_stream_load
```
Comment (jaogoy) [inline docs/zh/using_starrocks/query_acceleration_with_auto_increment.md:210]: http://

**Review by jaogoy — APPROVED** [review]

**Review by wangsimo0 — APPROVED** [review]

**Pull request merged**